pages/manual/contributing.md

```diff
@@ -32,3 +32,13 @@ It is recommended to use [ClangFormat](http://clang.llvm.org/docs/ClangFormat.ht
 
 ### Contributing whole modules
 If you are working on a large Inviwo Module and possibly have dependencies on big external libraries, we have an extra [modules repository](https://github.com/inviwo/modules).
+
+#### Inviwo Marketplace
+Additionally, there is the inviwo marketplace (see Screenshot) that allows developers to share their processors or shaders.
+So after you have generated the code template for a processor by following the [guide for generating processors](https://inviwo.org/manual-devguide-meta.html), you should create a repository on [GitHub](https://github.com/) based on our [template](link-to-template).
+
+##### Template
```

Comment:
Title: Inviwo github repository template?
Start with shortly explaining the purpose: Using this template will allow you to build your.... 
Then what it does and what one should do to use it: The template contains a workflow file....

Reply:
could we find a way to gather all these repos and build them when we update core, similar to the test we have when building core right now? Could some kind of "registration" be part of the template?

Reply:
For now the idea is to just have a file to register ones repo. Something like a json, providing a name, github url.. Using this file we could probably build an action that tests all registered modules. If one wants to submit their module to the "official" package index they would make a PR adding their module to this file. Reviewing this before merging will be rather manual for now I guess.

Reply:
Sounds resonable, just a suggestion looking at like how for example vcpkg handle that, it might be better to have a json file per registered module/repo. putting all in one file might make merges a bit more complicated...